src/editor/EditorStatusBar.js

```diff
@@ -226,6 +227,25 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Update LineEndings label
+     * @param {Event} event (unused)
+     * @param {Editor} editor Current editor
+     * @param {boolean=} doNotAnimate True if state should not be animated
+     */
+    function _updateLineEndingsLabel(event, editor, doNotAnimate) {
```

Comment:
I'm not sure this needs to be a separate function, since it's fairly simple and only has one caller